README.md

```diff
@@ -1,2 +1,65 @@
-# message-queue-toolkit
-Useful utilities, interfaces and base classes for message queue handling
+# message-queue-toolkit ✉️
+Useful utilities, interfaces and base classes for message queue handling.
+
+## Overview
+
+This is an abstraction to switch between different queue systems without having to implement your own deserialization, error handling, etc. The library provides utilities, interfaces and base classes to build the support for any queue system you may need in your service and already implements support for the following:
+
+* AMQP (Advanced Message Queuing Protocol)
```

Comment:
It would be helpful to mention which dependencies need to be installed for which protocol. See package.json for each subpackage for their names (e. g. `@message-queue-toolkit/sqs`)